cmd/kube-proxy/proxy.go

```diff
@@ -52,6 +53,7 @@ func init() {
 }
 
 func main() {
+	util.AddAllFlagsToPFlags()
```

Comment:
should we capture this and flag.Parse into util.InitFlags() to go with InitLogs() ?